### PR TITLE
Update get, list, update to use new model

### PR DESCRIPTION
**Pull request by tkyi**
Reflects changes made in model https://github.com/screwdriver-cd/models/pull/6

### CODE REVIEW

File: lib/update.js [hunk 1]
```diff
@@ -26,17 +27,16 @@ module.exports = (datastore) => ({
                 }
             };
 
-            datastore.update(config, (err, response) => {
+            Job.update(config, (err, response) => {
```
Comment (nkatzman) [inline lib/update.js]: I don't think this is the right object passed to `update`

Comment (d2lam) [inline lib/update.js]: config should only include id and data.

File: lib/update.js [hunk 2]
```diff
@@ -16,27 +16,24 @@ module.exports = (datastore) => ({
...
             const id = request.params.id;
             const data = request.payload;
             const config = {
-                table: 'jobs',
-                params: {
-                    id,
-                    data
-                }
+                id,
+                data
```
Comment (d2lam) [inline lib/update.js]: Can put `data: request.payload` here and get rid of the variable

File: README.md [hunk 3]
```diff
@@ -44,11 +44,15 @@ server.register({
 #### Updates a single job
 `PUT /jobs/{id}`
 
+**Arguments**
```
Comment (d2lam) [inline README.md]: Can we make this a smaller font? From the preview, "Updates a single job" and "Arguments" look like the same level.

**Comment by d2lam** [comment]
The rest LGTM 👍

File: README.md (hunk 3 above)
Comment (tkyi) [inline README.md]: 👍 I'll just make it normal size font. But we should fix the other plugin READMEs as well then

**Comment by nkatzman** [comment]
👍

**Pull request merged**